Podfile

```diff
@@ -1,11 +1,11 @@
-platform :ios, '12.0'
+platform :ios, '13.0'
```

Comment:
12.0 is too old, Concierge needs 12.4 minimum but since iOS is already on 18.2, I have used 13.0 as reference.